### PR TITLE
Use ~/.cql/ directory as default config location.

### PR DESCRIPTION
**Pull request by laodouya**
1. Change all cmd config param to ~/.cql/config.yaml
2. Config generate will default use ~/.cql/ as well.
3. Driver can reduce client.Init func call if expected using ~/.cql/
4. Update all unit test.
5. Update all README.md
6. Update client example.
7. Compatible with elder usage.
8. Skip compatibility test if branch name contains '/beta_'

### CODE REVIEW

File: client/_example/simple.go [hunk 1]
```diff
@@ -21,33 +21,27 @@ import (
...
 
 func main() {
 	log.SetLevel(log.InfoLevel)
-	var config, password, dsn string
+	var dsn string
 
-	flag.StringVar(&config, "config", "./conf/config.yaml", "config file path")
-	flag.StringVar(&dsn, "dsn", "", "database url")
-	flag.StringVar(&password, "password", "", "master key password for covenantsql")
+	flag.StringVar(&dsn, "dsn", "", "Database url")
```
Comment (ggicci) [inline client/_example/simple.go:32]: url -> URL is better

File: client/driver.go [hunk 2]
```diff
@@ -97,6 +103,18 @@ type ResourceMeta struct {
 	AdvancePayment uint64
 }
 
+func defaultInit() (err error) {
```
Comment (ggicci) [inline client/driver.go:106]: Why not use DSN parameters to pass config file path or master key values?

Comment (laodouya) [reply client/driver.go:106]: DSN is used for connecting CovenantSQL database by many users. It should not contain local file path that probably not found in other machine.

Comment (laodouya) [reply client/driver.go:106]: For now, our system and all command line series is designed to not shown master key. In that opinion, master key should not displayed in DSN parameters either.

**Review by auxten — APPROVED** [review]
LGTM

**Review by leventeliu — APPROVED** [review]
LGTM.

**Pull request merged**